jobapi/routes.go

```diff
@@ -0,0 +1,148 @@
+package jobapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/buildkite/agent/v3/agent"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+	"golang.org/x/exp/maps"
+)
+
+var protectedEnv map[string]struct{}
+
+func init() {
+	protectedEnv = make(map[string]struct{}, len(agent.ProtectedEnv))
+	for _, p := range agent.ProtectedEnv {
```

Comment:
Why isn't ProtectedEnv a `map[string]struct{}` already?

Reply:
🤔 excellent question. i hate using `init()` so i might just change the `agent.ProtectedEnv`